### DISCUSSION TITLE
required_indicator for form_label

### DISCUSSION THREAD

**Discussion by zulqasar** [reflex-dev/reflex#1743] [Q&A]
Chakra supports requiredIndicator for form_label. However setting this prop in reflex as no effect. Am I missing out something?

**Comment by masenf** [comment]
It looks like the Reflex FormLabel doesn't have that defined as a prop... in the meantime, you should be able to pass it using "custom_attrs" prop, which accepts a dict of props and applies them to the component whether they are defined or not.

**Reply by masenf** [reply]
At least one reason why this isn't implemented yet is that reflex doesn't support passing components as props yet; and looking closer at the docs, `requiredIndicator` and `optionalIndicator` are indeed expected to be passed as components. This has been in the backlog for a while, as #236, not sure when we'll be able to get to it...
